src/app/warren/settings.py

```diff
@@ -341,6 +342,25 @@ class Development(Base):
     DEBUG = True
     ALLOWED_HOSTS = ["*"]
 
+    # LTI parameters
+    LTI_PARAMETERS = {
+        "lti_message_type": "basic-lti-launch-request",
+        "lti_version": "LTI-1p0",
+        "resource_link_id": str(uuid.uuid4()),
+        "lis_person_contact_email_primary": Value(
+            "johndoe@example.com", environ_name="LIS_PERSON_CONTACT_EMAIL_PRIMARY"
+        ),
+        "user_id": Value("1234", environ_name="USER_ID"),
```

Comment:
```suggestion
        "user_id": Value("1234", environ_name="LTI_USER_ID"),
```

I suggest you to use `LTI_USER_ID`. `USER_ID` is really generic and you can probably have conflicts.